docs/cpe-authentication.rst

```diff
@@ -69,4 +69,29 @@ the credentials from an external source:
 ACS to CPE
 ----------
 
-TODO
+.. note::
+
+  By default, GenieACS does not have ACS to CPE authentication configured.
+
+The following parameters are used to set and get (password is redacted but
+can be set) the username/password used to authenticate against the CPE:
+
+Username: ``InternetGatewayDevice.ManagementServer.ConnectionRequestUsername``
+
+Password: ``InternetGatewayDevice.ManagementServer.ConnectionRequestUsername``
```

Comment:
update to  `InternetGatewayDevice.ManagementServer.ConnectionRequestPassword`

Reply:
This documentation only supports TR069 and not TR181. The correct syntax to support both is
```javascript
AUTH(username, password)
```

I would probably add a note in there that secondary auth params can be configured as well and GenieACS will try them in order until one works (or they all fail). Example:

```javascript
AUTH(username, password) OR AUTH("some_username", "some_password")
```

Reply:
line 81 did you mean  `InternetGatewayDevice.ManagementServer.ConnectionRequestUsername` or `InternetGatewayDevice.ManagementServer.ConnectionRequestPassword` ?

Reply:
No, line 97. And lines 79 and 81 should have `Device.ManagementServer.ConnectionRequestUsername` and `Device.ManagementServer.ConnectionRequestPassword` listed as the values that could be used.

ACS -> CPE auth is enabled by default in v1.2 and if the cwmp.connectionRequestAuth key is not set in the UI, then it will default to `AUTH(username, password)`

Here is a truncated version of the default inform script for v1.2.
``` javascript
// Device ID as user name
const username = declare("DeviceID.ID", {value: 1}).value[0]

// Password will be fixed for a given device because Math.random() is seeded with device ID by default.
const password = Math.trunc(Math.random() * Number.MAX_SAFE_INTEGER).toString(36);

// Refresh values daily
const daily = Date.now(86400000);

declare("InternetGatewayDevice.ManagementServer.ConnectionRequestUsername", {value: daily}, {value: username});
declare("InternetGatewayDevice.ManagementServer.ConnectionRequestPassword", {value: daily}, {value: password});

declare("Device.ManagementServer.ConnectionRequestUsername", {value: daily}, {value: username});
declare("Device.ManagementServer.ConnectionRequestPassword", {value: daily}, {value: password});
```